src/Cms/Auth.php

```diff
@@ -493,23 +479,19 @@ public function status($session = null, bool $allowImpersonation = true)
     }
 
     /**
-     * Validates the user credentials and returns the user object on success;
-     * otherwise logs the failed attempt
+     * Ensure that email addresses with IDN domains are in Unicode format
+     * and that rate limit was not exceeded
      *
      * @param string $email
-     * @param string $password
-     * @return \Kirby\Cms\User
+     * @return string
      *
-     * @throws \Kirby\Exception\PermissionException If the rate limit was exceeded or if any other error occurred with debug mode off
-     * @throws \Kirby\Exception\NotFoundException If the email was invalid
-     * @throws \Kirby\Exception\InvalidArgumentException If the password is not valid (via `$user->login()`)
+     * @throws \Kirby\Exception\PermissionException If the rate limit was exceeded
      */
-    public function validatePassword(string $email, string $password)
+    public function validateEmail(string $email): string
     {
         // ensure that email addresses with IDN domains are in Unicode format
         $email = Idn::decodeEmail($email);
 
-        // check for blocked ips
```

Comment:
I think this comment is quite helpful. Probably was removed because the first occurrence of this duplicated block didn't contain the comment for whatever reason.

Reply:
Oh yes, good catch, will add it back